src/java/com/zimbra/cs/nginx/ServerInfo.java

```diff
@@ -58,7 +60,7 @@ void setImapSSLPort(String port) {
         mHttpImapSSLPort = port;
     }
     
-    String getPortForProto(String proto, boolean isZimbraAdmin) {
+    String getPortForProto(String proto, boolean isZimbraAdmin, boolean isZimbraZx) {
```

Comment:
Does is makes sense to program this function using single parameter that decides on ZimbraAdmin ot ZimbraZx.